wolfcrypt/benchmark/benchmark.c

```diff
@@ -2146,53 +2300,71 @@ static void bench_stats_asym_finish_ex(const char* algo, int strength,
         opsSec = 0;
     }
 
+#ifdef BENCH_MICROSECOND
+    milliEach = each / 1000;   /* milliseconds */
+#else
     milliEach = each * 1000;   /* milliseconds */
 
+#endif
+
+#ifdef BENCH_MICROSECOND
```

Comment:
You can move this up to the declaration: turn `digits` into a `const int`, and conditionalize the initializer on `BENCH_MICROSECOND`.  Not important, but it's neater and more efficient.